### PR TITLE
Icepak mesh

### PR DESCRIPTION
**Pull request by boyang2022**

### CODE REVIEW

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -45,6 +45,7 @@ test = [
     "mypy==0.960",
     "pytest==6.2.5",
     "pytest-cov==3.0.0",
+    "psutil==5.9.1",
```
[the remark this answers is not in the document]
Comment (boyang2022) [reply pyproject.toml:48]: also added in test reqs

File: aedttest/simulation_data.py [hunk 2]
```diff
@@ -270,16 +275,19 @@ def extract_design_data(app, design_name, setup_dict, project_dir, design_dict):
...
-            mesh_data = parse_mesh_stats(mesh_stats_file, design_name, variation_name, setup)
-            design_dict[design_name]["mesh"][variation_name][setup] = mesh_data
-
             profile_file = generate_unique_file_path(project_dir, ".prof")
             profile_file = app.export_profile(setup, variation_string, profile_file)
-            simulation_time = parse_profile_file(profile_file, design_name, variation_name, setup)
+            simulation_time, cell_number = parse_profile_file(profile_file, design_name, variation_name, setup)
             design_dict[design_name]["simulation_time"][variation_name][setup] = simulation_time
 
+            if app.design_type == "Icepak":
```
[the remark this answers is not in the document]
Comment (boyang2022) [reply aedttest/simulation_data.py:283]: no, it has mesh in the profile

File: pyproject.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (beliaev-maksim) [reply pyproject.toml:48]: Why is it here?
Have you seen any CPy test failing because of it?

File: tests/reqs_test_sim_data_py27.txt [hunk 3]
```diff
@@ -3,4 +3,5 @@ pytest-cov==2.12.1
 mock==3.0.5
 pyaedt
 Django
-pywin32==228
\ No newline at end of file
+pywin32==228
+psutil==5.9.1
```
Comment (beliaev-maksim) [inline tests/reqs_test_sim_data_py27.txt:7]: Add also one empty line at the end of the file

File: pyproject.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (boyang2022) [reply pyproject.toml]: removed

File: tests/reqs_test_sim_data_py27.txt (hunk 3 above)
Comment (boyang2022) [reply tests/reqs_test_sim_data_py27.txt]: done

**Review by beliaev-maksim — APPROVED** [review]

**Pull request merged**